src/traits.rs

```diff
@@ -1510,6 +1520,13 @@ pub trait ComponentExt: Debug {
     ///     println!("{:?}°C", component.critical());
     /// }
     /// ```
+    ///
+    /// ## Linux hwmon
+    ///
+    /// Critical threshold defined by chip or kernel may be defined or not.
+    /// Implementation detail:
+    ///
+    /// `thresold_critical` is >= `thresold_max`.
```

Comment:
```suggestion
    /// ## Linux
    ///
    /// Critical threshold defined by chip or kernel.
    ///
    /// Implementation detail:
    ///
    /// `critical` >= `max`.
```

Reply:
Not totally agree on this one. `max` in terms of `sysinfo is not `max` in term of linux way of viewing things, since `max_thresold` aka `linux temp1_max` is not exposed. It's a bit of a leakage of information.